File: CPP/8/src/Student.cpp
/*
 * Student.cpp
 *
 *  Created on: 17-Aug-2024
 *      Author: Manish Patil
 */

#include "Student.h"
#include<string>
using namespace std;

void newStudent(StudentD* s,string fn, string ln, int roll, int fM){
	s->firstName=fn;
	s->lastName=ln;
	s->roll=roll;
	s->fMarks=fM;
}

string getName(StudentD* s){
	string name=s->firstName;
	name.append(" ");
	name.append(s->lastName);
	return name;
}

int getRoll(StudentD* s){return s->roll;}
int getFMarks(StudentD* s){return s->fMarks;}

File: Java/com/manish/assignments/eight/Student.java
package com.manish.assignments.eight;

public class Student {
	private String name;
	private int roll;
	private int marks;
	
	Student(String name){
		this.name=name;
	}
	
	public void setRoll(int m) {this.roll=m;}
	public int getRoll() {return this.roll;}
	
	public void setMarks(int m) {this.marks=m;}
	public int getMarks() {return this.marks;}
	
	public String getName() {return this.name;}
}
